mart/configs/experiment/ArmoryCarlaOverObjDet_TorchvisionFasterRCNN.yaml

```diff
@@ -2,6 +2,7 @@
 
 defaults:
   - COCO_TorchvisionFasterRCNN
+  - override /model/detection@model.modules.preprocessor: preprocessor_multi_modal
```

Comment:
Why is this the default now?

Reply:
A preprocessor is required and I made the single-modal normalizer the default one.

Reply:
But that specifies the multimodal one?